js/app/restaurant/mock-restaurant.provider.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Provides data from a json file that was generated from the previous site
```

Comment:
This could probably use more elaboration, but really I'm highlighting this because the other JS files need headers like this.

Reply:
could you give another example? i think if you know angular most of these should be pretty clear based on the file naming.

Reply:
I suppose you're right.  As I mentioned in my overall comment, this isn't a deal-breaker.